src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPalExpress.class.php

```diff
@@ -115,7 +115,7 @@ public function PostProcessExternalPaymentHandlerHook()
      *
      * @return void
      */
-    protected function GetUserDataFromPayPalData(&$aBilling, &$aShipping)
+    protected function GetUserDataFromPayPalData($aBilling, $aShipping)
```

Comment:
Those should stay as well

Reply:
good catch - and the method called from within this one then as well (`\TShopPaymentHandlerPayPalExpress::postProcessBillingAndShippingAddress`)